packages/suite/src/components/firmware/OnboardingStepBox.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ConfirmOnDevice } from '@trezor/components';
+import { Box, BoxProps } from '@onboarding-components';
+import { Backdrop, Translation } from '@suite-components';
+
+const OuterActions = styled.div`
+    display: flex;
+    margin-top: 40px;
+    width: 100%;
+    justify-content: center;
+`;
+
+const InnerActions = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 32px;
+`;
+
+const ConfirmWrapper = styled.div`
+    margin-bottom: 20px;
+    z-index: 1;
+    height: 62px;
+`;
+
+interface Props extends BoxProps {
+    innerActions?: React.ReactNode;
+    outerActions?: React.ReactNode;
+    confirmOnDevice?: number;
+    disableConfirmWrapper?: boolean;
+}
+
+const OnboardingStepBox = ({
+    heading,
+    description,
+    image,
+    innerActions,
+    outerActions,
+    confirmOnDevice,
+    disableConfirmWrapper,
+    className,
+    children,
+    ...rest
+}: Props) => (
+    <>
+        <Backdrop show={!!confirmOnDevice} animated zIndex={0} />
+        {!disableConfirmWrapper ? (
+            // todo: hey why is this file under /firmware path? :D
+            <ConfirmWrapper data-test="@onboarding/confirm-on-device">
+                {typeof confirmOnDevice === 'number' ? (
```

Comment:
Why not check for `confimDevice` being defined? It is already typed as a number. And this could also be joined to the condition above.

Reply:
0 is number to and it wouldn't render anything, but yeah doesn't really matter since there is no trezor 0, but still :)

Reply:
It could also default to 0 using `defaultProps` and the condition could be `confirmDevice > 0`.